qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -1 +1,40 @@
-Feature: Visit types Management
\ No newline at end of file
+Feature: Visit types Management
+
+  Background:
+    Given a user clicks on the system administration link from the home page
+    Then the administration page gets loaded
+    And a user clicks on the manage Visit Types link
+    Then the manage visit types page is loaded
+
+  @selenium
+  @visitType
+  Scenario: Adding a Visit Type
+    When a user clicks on the add visit type link on the manage visit type page
+    Then the system loads the visit type form
+    And a user fills the add visit type form
+    And a user saves visit type
+    Then a visit type is saved successfully
+
+  @selenium
+  @visitType
+  Scenario: Editing a Visit Type Test
+    When a user clicks on the already saved visit type link
+    Then the system loads the visit type form
```

Comment:
```js
When a user clicks on the already saved visit type link
Then the system loads the visit type form
```
These steps would be considered under Background since they are being used by all the Scenarios

Reply:
but @kdaud `When a user clicks on the already saved visit type link` its not in the` Adding a Visit Type` Scenario